cmd/snap-update-ns/main.go

```diff
@@ -42,13 +44,19 @@ func main() {
 func parseArgs(args []string) error {
 	parser := flags.NewParser(&opts, flags.HelpFlag|flags.PassDoubleDash|flags.PassAfterNonOption)
 	_, err := parser.ParseArgs(args)
-	return err
+	if err != nil {
```

Comment:
This could be written as a single line: `if _, err := parser.ParseArgs(args); err != nil {...`